components/raftstore/src/store/fsm/store.rs

```diff
@@ -374,6 +390,21 @@ where
         }
         timeout
     }
+
+    pub fn update_ticks_timeout(&mut self) {
```

Comment:
Should call this function after config updated at https://github.com/tikv/tikv/blob/7ce49e7d2d2a8ef34bd54b43523df0790ba7b1df/components/raftstore/src/store/fsm/store.rs#L727-L747

or instead of maintaining `wait_duration`, we can get the up-to-date config from `self.cfg` through a function like `fn get_wait_duration(&self, tick: PeerTicks) -> Duration`

Reply:
Ok.